.devcontainer/local/docker-compose.yaml

```diff
@@ -0,0 +1,13 @@
+version: "3"
+services:
+  app:
+    cpus: 4
+    mem_limit: 8g
```

Comment:
Are these enough resources to run `cellpose` on the example data?  How long does it take to run?

Reply:
If it is enough, it will take a long time. I've been running the notebook outside the devcontainer using my machine's resources and it takes ~120 minutes for the current example dataset.

Reply:
Wow.  What are the dimensions of the volume and `cellpose` settings?  Perhaps we can change these to reduce the compute time.

Reply:
I have an update on this - I ran a smaller stack with ~ 100 frames and it cut down the time in `cellpose` to ~ 15 minutes. This might still be too long but there isn't much else we can change within `cellpose` to reduce the time further from what I can tell.